test/ui/toggle_3ds_ui_tests.js

```diff
@@ -296,9 +313,38 @@ describe('The turn 3D Secure on confirmation page', function () {
         toggle_3ds_update: false
       }
     }
-
     let body = renderTemplate('3d_secure/on_confirm', templateData)
 
     body.should.containNoSelector('#3ds-confirm-on-button')
   })
+
+  it('should not display helper test if not worldpay', function () {
+    let templateData = {
+      'supports3ds': true,
+      'showHelper3ds': false,
+      permissions: {
+        toggle_3ds_read: true,
+        toggle_3ds_update: true
+      }
+    }
+
+    let body = renderTemplate('3d_secure/on_confirm', templateData)
+
+    body.should.containNoSelector('#3ds-helper-text-worldpay')
+  })
+
+  it('should display helper test if worldpay', function () {
+    let templateData = {
```

Comment:
We can use `const` by default for local and global variables. In case the variable will be changed (with another reference) we can use `let` and `var` is not used